crates/transaction-emitter-lib/src/args.rs

```diff
@@ -54,16 +54,19 @@ pub struct ClusterArgs {
     pub mint_args: MintArgs,
 }
 
-#[derive(Debug, Clone, Copy, ArgEnum, Deserialize, Parser, Serialize)]
+#[derive(Debug, Clone, Copy, ArgEnum, Deserialize, Parser, Serialize, PartialEq, Eq, Hash)]
 pub enum TransactionType {
+    // Deprecate P2P name
```

Comment:
Why deprecate P2P? `Transfer` is ambiguous as it can mean any type of transfer of asset including NFTs. P2P name is more appropriate for coin transfer - may be we can add a new enum `P2PHighPri` to complement it?

Reply:
P2P can mean anything too? (i.e. peer-to-peer or person-to-person sending of anything)

TransferCoins?